Pandas_Introduction.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>Lets add another section below this that uses the ipython <code>display</code>  method.</p><p> </p><p><code>display(data_frame)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Should be included!